tests/test_runs.py

```diff
@@ -157,13 +163,19 @@ def test_run_forward(existing_temp_model, monkeypatch, setup_deps):
     delta_qoi = slvr.Qval_ts[-1] - slvr.Qval_ts[0]
     u_norm = norm(slvr.U)
 
+    if pytest.parallel:
+        tol = 1e-3
+
+    else:
+        tol = 1e-5
+
     pytest.check_float_result(delta_qoi,
                               expected_delta_qoi,
-                              work_dir, 'expected_delta_qoi', tol=1.0e-5)
```

Comment:
should this be tol=tol since it is already set above?

Reply:
@dngoldberg it is .. you are looking at the red changes are what I changed... in green you can see that `tol=tol` see line 174